frontend/src/screens/channels/Channels.tsx

```diff
@@ -506,8 +508,18 @@ export default function Channels() {
                         </div>
                       </TooltipTrigger>
                       <TooltipContent className="w-[300px]">
-                        Your spending balance is the funds on your side of your
-                        channels, which you can use to make lightning payments.
+                        Your spending balance is the spendable funds on your
+                        side of your channels, which you can use to make
+                        lightning payments.{" "}
+                        {info?.backendType === "LND" &&
```

Comment:
I am wondering if we should always show this info, then users running LDK also can see the reserved amount and total on this tooltip?

Reply:
I just discovered that we don't remove the reserves in LDK.

See `LocalBalance` calculation:
https://github.com/getAlby/hub/blob/eaf72825c0001dfacab97d7bcb22dffa122dfd82/lnclient/ldk/ldk.go#L858

And see `totalSpendable` calculation:
https://github.com/getAlby/hub/blob/eaf72825c0001dfacab97d7bcb22dffa122dfd82/lnclient/ldk/ldk.go#L1638

(Similarly we don't exclude the reserves in `totalReceivable` as well)

Reply:
`LocalBalance` should include the reserve based on the current logic. `LocalSpendableBalance` does not have the reserves.